pygmt/gridops.py

```diff
@@ -113,3 +113,115 @@ def grdcut(grid, **kwargs):
             result = None  # if user sets an outgrid, return None
 
         return result
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    F="filter",
+    D="distance",
+    I="increment",
+    N="nans",
+    R="region",
+    T="toggle",
+    V="verbose",
+    f="colinfo",
+)
+@kwargs_to_strings(R="sequence")
+def grdfilter(grid, **kwargs):
+    """
+    filter a grid file in the time domain using one of the selected convolution
+    or non-convolution isotropic or rectangular filters and compute distances
+    using Cartesian or Spherical geometries. The output grid file can optionally
+    be generated as a sub-region of the input (via *region*) and/or with new increment
+    (via *spacing*) or registration (via *toggle*).
+    In this way, one may have “extra space” in the input data so that the edges
+    will not be used and the output can be within one half-width of the input edges.
+    If the filter is low-pass, then the output may be less frequently sampled than the input.
```

Comment:
```suggestion
    Filter a grid in the space (or time) domain.

    Filter a grid file in the time domain using one of the selected convolution
    or non-convolution isotropic or rectangular filters and compute distances
    using Cartesian or Spherical geometries. The output grid file can
    optionally be generated as a sub-region of the input (via *region*) and/or
    with new increment (via *spacing*) or registration (via *toggle*). In this
    way, one may have "extra space" in the input data so that the edges will
    not be used and the output can be within one half-width of the input edges.
    If the filter is low-pass, then the output may be less frequently sampled
    than the input.

    Full option list at :gmt-docs:`grdfilter.html`

    {aliases}
```
Adds a link to full documentation at https://docs.generic-mapping-tools.org/6.1/grdfilter.html, include aliases, and wraps to 79 characters to conform with flake8 style (see also #384).